arelle/Cntlr.py

```diff
@@ -18,15 +18,18 @@
 import subprocess
 import sys
 import tempfile
-from collections import defaultdict
-from typing import Any, Mapping, TYPE_CHECKING, TextIO, cast
+from typing import Any, TYPE_CHECKING, cast
 
 import regex as re
 
 from arelle import Locale, ModelManager, PackageManager, PluginManager, XbrlConst
 from arelle.BetaFeatures import BETA_FEATURES_AND_DESCRIPTIONS
 from arelle.SystemInfo import PlatformOS, getSystemWordSize, hasFileSystem, hasWebServer, isCGI, isGAE
 from arelle.WebCache import WebCache
+from arelle.logging.formatters import LogFormatter as _LogFormatter
```

Comment:
Is there a reason this won't work?
```python
from arelle.logging.formatters.LogFormatter import LogFormatter, logRefsFileLines
```

Reply:
Just that the current approach is more explicit about making reimport available, and that it prevents import cleanup from affecting this line. Though I suppose a `# noqa - for reimport` comment does the same thing.

Reply:
Add a small test to make sure you can import these items from `Cntlr.py`?